pkg/cli/flags.go

```diff
@@ -266,6 +266,8 @@ func init() {
 		// TODO(peter): Decide if we want to make the lightstep flags visible.
 		if strings.HasPrefix(flag.Name, "lightstep_") {
 			flag.Hidden = true
+		} else if flag.Name == "no-redirect-stderr" {
+			flag.Hidden = true
```

Comment:
I think a separate flag is useful for debugging logging issue (for us). I also think it's sufficiently esoteric that we want to keep it hidden from docs indeed.